cloud/pkg/cloudstream/streamserver.go

```diff
@@ -228,7 +217,11 @@ func (s *StreamServer) getExec(request *restful.Request, response *restful.Respo
 		}
 	}()
 
-	sessionKey := strings.Split(request.Request.Host, ":")[0]
+	sessionKey, err := s.getSessionKey(request.Request.URL.Path)
+	if err != nil {
+		err = fmt.Errorf("can not get session key: %v", err)
```

Comment:
ditto

Reply:
done. I have modifed error log message in defer clause